src/yolov7_reid/src/detector.py

```diff
@@ -183,34 +183,34 @@ def detection_with_image(frame, display_in_queue=True):
         combined_img = yolov7_detector.draw_detections_with_predefined_colors(frame,person_bboxes, person_scores, person_class_ids,pre_colors)
 
         send_image = False
-        if unknown > 0:
+        if unknown > 0 and unknown ==1:
             send_image = True
-            if unknown == 1:
-                telegram_bot.send('SharpAI saw one unfamiliar person')
+            telegram_bot.send('SharpAI saw one unfamiliar person')
+            
             else:
```

Comment:
If moved unknown == 1 to upper layer, then this `else` will be wrong, need to fix.

Reply:
Done.